cloudbuild.yaml

```diff
@@ -127,21 +133,28 @@ steps:
       - |
         set -e
 
-        cd jupyter-on-gke/
+        cd /workspace/applications/jupyter/
+        terraform destroy \
+        -var-file=workloads-without-iap.example.tfvars \
+        -var=project_id=$PROJECT_ID \
+        -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-cluster \
+        -var=namespace=ml-$SHORT_SHA \
+        -var=gcp_and_k8s_service_account=jupyter-sa-$SHORT_SHA \
+        -var=gcs_bucket=gke-aieco-jupyter-$SHORT_SHA \
+        -auto-approve -no-color
+
+        cd /workspace/applications/ray/
         terraform destroy \
+        -var-file=workloads.tfvars \
         -var=project_id=$PROJECT_ID \
-        -var=namespace=$SHORT_SHA \
-        -var=add_auth=false \
-        -var=enable_iap_service=false \
-        -auto-approve -no-color 
-
-        cd ../ray-on-gke/user/
-        terraform destroy -var=project_id=$PROJECT_ID \
-        -var=namespace=$SHORT_SHA \
-        -var=service_account=$_USER_NAME-$SHORT_SHA-system-account \
+        -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-cluster \
+        -var=cluster_location=$_REGION \
+        -var=ray_namespace=ml-$SHORT_SHA \
+        -var=gcp_service_account=ray-sa-$SHORT_SHA \
+        -var=gcs_bucket=gke-aieco-ray-$SHORT_SHA \
         -auto-approve -no-color
 
-        gcloud container clusters delete ray-$SHORT_SHA-$_PR_NUMBER-cluster --zone $_ZONE  --project $PROJECT_ID --quiet
+        gcloud container clusters delete ml-$SHORT_SHA-$_PR_NUMBER-cluster --location $_REGION  --project $PROJECT_ID --quiet
```

Comment:
We should delete the cluster using Terraform destroy from infrastructure/ as 'create gke cluster' step above may create additional resources.

Reply:
I see that it's a variable now so we can disable it and use terraform to delete resources, I'll update to use that